src/app/components/Header.tsx

```diff
@@ -1,60 +1,108 @@
-    { label: "TEAM", href: "#team" },
-    { label: "CONTACT US", href: "#contact-us" },
+    { label: 'HOME', href: '#home' },
+    { label: 'EVENTS', href: '#events' },
+    { label: 'TEAM', href: '#team' },
+    { label: 'CONTACT US', href: '#contact-us' },
   ];
 
   return (
     <AppBar
       position="static"
       sx={{
-        backgroundColor: "#0A001F",
+        backgroundColor: '#0A001F',
         height: 80,
-        marginLeft: "auto",
-        marginRight: "auto",
-        maxWidth: "1000px",
-        width: "100%",
+        maxWidth: '1000px',
+        margin: '20px auto',
         borderRadius: 2,
-        marginTop: "20px",
       }}
     >
-      <Toolbar
-        sx={{
-          justifyContent: "space-between",
-          minHeight: 80,
-          paddingX: 3,
-        }}
-      >
-        {/* Logo Section */}
-        <Box sx={{ display: "flex", alignItems: "center" }}>
-          <Image src="/assets/icon.svg" alt="Logo" width={110} height={110} />
-        </Box>
+      <Toolbar sx={{ justifyContent: 'space-between', minHeight: 70, px: 3 }}>
+        {/* Logo */}
+        <Image src="/assets/icon.svg" alt="Logo" width={150} height={150} />
 
-        {/* Navigation Links */}
-        <Box sx={{ display: "flex", gap: 4 }}>
+        {/* Desktop Navigation */}
+        <Box sx={{ display: { xs: 'none', md: 'flex' }, gap: 4 }}>
           {navItems.map((item) => (
             <Link key={item.label} href={item.href} passHref>
               <Button
                 sx={{
-                  color: "white",
-                  fontWeight: "bold",
-                  fontSize: "14px",
-                  "&:hover": { color: "#BDBDBD" },
+                  color: '#FFFFFF',
+                  fontWeight: 'bold',
+                  fontSize: '14px',
                 }}
               >
                 {item.label}
               </Button>
             </Link>
           ))}
         </Box>
+
+        {/* Mobile Menu Button */}
+        <IconButton
+          sx={{
+            display: { xs: 'flex', md: 'none' },
+            color: '#FFFFFF',
+            position: 'absolute',
+            top: 20,
+            right: 10,
+          }}
+          onClick={() => setDrawerOpen(!isDrawerOpen)}
+        >
+        {isDrawerOpen?<CloseIcon sx={{ fontSize: '30px'}}/>:<MenuIcon sx={{ fontSize: '30px' }} />}
+        </IconButton>
+
+          <Drawer
+          anchor="right"
+          open={isDrawerOpen}
+          onClose={()=>setDrawerOpen(false)}
+          sx={{
+            '& .MuiDrawer-paper': {
+              backgroundColor: 'rgba(10, 0, 31, 0.85)',
+              color: '#FFFFFF',
+              padding: '5px',
+              top: 90,
+              right: 5,
+              width: 'auto',
+              maxWidth: '250px',
+              height: 'auto',
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+              borderRadius: '10px',
```

Comment:
These styles has no effect on the ui so you can remove this
```suggestion
            '& .MuiDrawer-paper': {
              backgroundColor: 'rgba(10, 0, 31, 0.85)',
              color: '#FFFFFF',
              padding: '5px',
              top: 90,
              right: 5,
              height: 'auto',
              borderRadius: '10px',
```